packages/x-components/src/utils/types.ts

```diff
@@ -197,3 +197,14 @@ export interface DebounceOptions {
 export interface ThrottleFunction<Params extends any[]> {
   (...args: Params): void;
 }
+
+/**
+ * A list of next queries.
+ *
+ * @public
+ */
+// TODO Remove this interface when the original from the next-queries.list.vue is exported.
+export interface NextQueriesGroup extends ListItem {
```

Comment:
Why duplicate it? why not export the type in `NextQueriesList` component and use it?

Reply:
We need to do it in another PR, release and alpha an import it using the library, right?.

Reply:
Why???, they are both types from the same package, even if this type belonged to another package of the mono-repo it could be done in this PR, that's the whole point of the mono-repo.

Reply:
![hack-khaby](https://user-images.githubusercontent.com/5759712/136966948-8b838a94-46c2-47c3-ad51-7620722c965c.gif)